src/board.rs

```diff
@@ -115,6 +115,39 @@ impl Chessboard {
         self.white_turn = true;
     }
 
+    pub fn one_side_pieces(cb: &Chessboard, white: bool) -> u64 {
```

Comment:
make  these take `&self` as parameters instead of chessboard objects.

Reply:
done